MXParallaxHeader/MXParallaxHeader.m

```diff
@@ -214,7 +214,13 @@ - (void)setBottomModeConstraints {
 
 - (void)layoutContentView {
     CGFloat minimumHeight = MIN(self.minimumHeight, self.height);
-    CGFloat relativeYOffset = self.scrollView.contentOffset.y + self.scrollView.contentInset.top - self.height;
+    CGFloat topInset;
+    if (@available(iOS 11.0, *)) {
+        topInset = self.scrollView.adjustedContentInset.top;
```

Comment:
if you add 
            topInset = self.scrollView.contentInsetAdjustmentBehavior = UIScrollViewContentInsetAdjustmentNever;
before that, it will work with translucent navigationBar